src/python/pants/init/BUILD

```diff
@@ -17,7 +18,6 @@ target(
         "src/python/pants/backend/experimental/codegen/thrift/scrooge/java",
         "src/python/pants/backend/experimental/codegen/thrift/scrooge/scala",
         "src/python/pants/backend/experimental/debian",
-        "src/python/pants/backend/experimental/docker",
```

Comment:
Bad change! This still needs to be there.

Reply:
Oh, right. Now I get what you mean, as it's still present, but deprecated.